src/templates/subscribe/index.ts

```diff
@@ -6,7 +6,8 @@ export default async function prepare(params: TemplatePrepareParams) {
   const verifyLink = `${process.env.FRONT_HOST}/#/verify?${new URLSearchParams({
     signature: await signVerify(params.to, params.address, params.salt),
```

Comment:
Here we have a type-matching problem. If you check `typeof params.salt` at this line you will get `number`. But since we send it as a part of the URL query, it will implicitly be converted to a string, and at `envelop-ui` project the `typeof route.query.salt` will return the string. Respectively during `verifyVerify` function we always will get `false` since we signed the message with salt type `number`, but verifying it with salt type `string`